### ISSUE TITLE
Implement Wasp CLI Powerline

### ISSUE THREAD

**Issue by Martinsos** [wasp-lang/wasp#250]
Coming from #207.

Idea is to provide some kind of nice progress indicator in Wasp CLI. Idea: at the bottom of the screen, there could be a "readonly" line that shows special "wasp status" -> it would be showing current status of the `wasp start`. So while we are printing stuff, that line would always be there. While packages are installing, it could be showing smth like [1/2] Installing web-app and server .... Then, once that is done, it could show something like `All good - wasp is running in dev mode`!, and if at some point server crashes or smth like that, it could be showing `Server broken - fix it`. It could also be showing stuff like `schema is out of sync with db - run 'wasp db migrate-dev' to sync it`.

There is cool package on npm that might server as inspiration https://www.npmjs.com/package/ora .
This is however something we should be doing in Haskell!

**Comment by turbodami** [comment]
Cool one!

**Comment by Martinsos** [comment]
A spin on this:
When developing wasp app, in your browser's tab, you would see a small something that would be showing the status of wasp start -> are there any warnings or errors. It could be a small circle, that is green if all ok, orange if warnings, red if errors. Maybe you could move it around if it is bothering you. It could start spinning when wasp start is compiling.
This would help with the fact that Wasp might print a warning in the CLI, but you won't see it!
Or maybe this could even be shown in your editor? But that would be editor dependent.

**Comment by Martinsos** [comment]
Alternative is to not have a CLI powerline, but instead make all this part of Wasp Studio (client/server/db logs, warnings and errors, ...).

**Comment by Martinsos** [comment]
Yeah, we could have Wasp Studio which is a hub for everything: info about your app, its parts, about all the logs that are coming from it, and also for interacting with it.

Related issue (triggering operations / jobs / apis): https://github.com/wasp-lang/wasp/issues/2343 .
